pylzy/lzy/storage/async_/fs.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+from typing import BinaryIO, Optional, Callable, Any
+import os
+import shutil
+from urllib.parse import urlparse
+
+from lzy.storage.api import AsyncStorageClient
+
+
+class FsStorageClient(AsyncStorageClient):
+    async def size_in_bytes(self, uri: str) -> int:
+        path = self.__path_from_uri(uri)
+        return os.path.getsize(path)
```

Comment:
It's not a recursive size, it's just size of folder itself.

Reply:
This client aims to work with individual blobs (files), not folders.